cli/interface.go

```diff
@@ -0,0 +1,215 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"text/tabwriter"
+
+	"github.com/hyperhq/runv/api"
+	"github.com/hyperhq/runv/hypervisor"
+	"github.com/urfave/cli"
+)
+
+var interfaceCommand = cli.Command{
+	Name:  "interface",
+	Usage: "manage interfaces for container",
+	Subcommands: []cli.Command{
+		infAddCommand,
+		infRmCommand,
+		infUpdateCommand,
+		infListCommand,
+	},
+	Before: func(context *cli.Context) error {
+		return cmdPrepare(context, true, context.Bool("detach"))
```

Comment:
Should context.Bool("detach")  be changed to true?

Reply:
Sure. I'll do it.